projectile-rails.el

```diff
@@ -204,6 +204,20 @@
 (defvar-local projectile-rails-zeus-sock nil
   "The path to the Zeus socket file")
 
+(defvar projectile-rails-generators
+  '(("assets" (("app/assets/javascripts/" "app/assets/javascripts/\\(.+\\)\\.coffee$")))
```

Comment:
@a3ammar I guess `"assets"` is one of the possible completion. But what's the idea behind `(("app/assets/javascripts/" "app/assets/javascripts/\\(.+\\)\\.coffee$"))`? And why only `coffee`?
Sorry I did try to find information on that but I failed.